plugins/org.jkiss.dbeaver.launcher/src/org/eclipse/equinox/launcher/Main.java

```diff
@@ -1865,6 +1869,30 @@ private String getProductVersion() {
         return appVersion;
     }
 
+    private String getProductId() {
```

Comment:
Copy-paste of getProductVersion
Use single function for product info read

Reply:
Rework to one line